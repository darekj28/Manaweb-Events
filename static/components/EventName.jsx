var React = require('react');
export default class EventName extends React.Component{
	render() {
		return(<div id="EventName" className="pull-left"><h1>{this.props.name}</h1></div>);
	}
}