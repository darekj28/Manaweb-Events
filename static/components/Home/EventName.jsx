var React = require('react');
export default class EventName extends React.Component{
	render() {
		return(

			<div id="EventName" className="pull-left">
				{this.props.numUnseenPosts == -1 ?
					<h1>{this.props.name}  </h1> :
					<h1>{this.props.name} <small> {this.props.numUnseenPosts} unseen posts </small> </h1> 
				}
			</div>);
	}
}