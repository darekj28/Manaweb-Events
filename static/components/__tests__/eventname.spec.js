import EventName from "../EventName.jsx";

describe("EventName", () => {
	it("renders correctly", () => {
		const en = renderer.create(
			<EventName name="Jay Kim"/>
			);
		expect(en).toMatchSnapshot();
	});
});