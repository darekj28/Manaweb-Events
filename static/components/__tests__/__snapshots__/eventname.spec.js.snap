exports[`EventName renders correctly 1`] = `
<div
  className="pull-left"
  id="EventName">
  <h1>
    Jay Kim
  </h1>
</div>
`;
